lms/static/sass/features/_course-experience.scss

```diff
@@ -203,3 +203,5 @@
 .course-reviews-tool {
   margin: ($baseline * 2) ($baseline * 3);
 }
+
+@import '../stanford/_sneakpeek_courseware.scss';
```

Comment:
Why the new include?

Reply:
cause this is a new SASS file that styles the new course overview page.